### PR TITLE
William/delete account

### PR DESCRIPTION
**Pull request by swansontec**
The visuals are garbage here - we really need to turn this into a scene and fix the scrolling. However, it does work.

Adding https://github.com/EdgeApp/edge-login-ui-rn/pull/2 will make the username list look correct after the delete is complete.

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201725606633205

### CODE REVIEW

File: src/components/themed/ThemedModal.js [hunk 1]
```diff
@@ -19,26 +19,33 @@ type Props<T> = {
...
  */
 export const ThemedModal = <T>(props: Props<T>) => {
-  const { bridge, children = null, flexDirection, iconRem = 0, justifyContent, onCancel } = props
+  const { bridge, children, flexDirection, iconRem = 0, justifyContent, warning = false, onCancel } = props
   const paddingRem = fixSides(props.paddingRem, 1)
   const theme = useTheme()
 
   paddingRem[0] += iconRem / 2
 
+  // TODO: The warning styles are incorrectly hard-coded:
```
Comment (samholmes) [inline src/components/themed/ThemedModal.js:38]: Mention the relation to the login UI (inherited styles from there)

**Review by samholmes — APPROVED** [review]
Add comment then merge

File: src/components/services/EdgeCoreManager.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ const contextOptions: EdgeContextOptions = {
     }
   },
 
-  plugins: allPlugins
+  plugins: allPlugins,
+  authServer: 'https://login-test.edge.app/api'
```
Comment (samholmes) [inline src/components/services/EdgeCoreManager.js:38]: Remove this and merge, or make it configurable from `env.json` (alternatively, `src/theme/appConfig.js`

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See the latest comment from sam @samholmes

**Pull request merged**